todo.txt

```diff
@@ -0,0 +1,14 @@
+1) commit con layout base 1 colonna centrata 
```

Comment:
usually TODO files are not commited in the repository as they are not a part of the project, it's just something usefull for the programmer